src/test/java/ch/wisv/areafiftylan/integration/WebBannerIntegrationTest.java

```diff
@@ -0,0 +1,300 @@
+package ch.wisv.areafiftylan.integration;
+
+import ch.wisv.areafiftylan.users.model.User;
+import ch.wisv.areafiftylan.web.banner.model.Banner;
+import ch.wisv.areafiftylan.web.banner.service.BannerRepository;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.restassured.http.ContentType;
+import org.apache.http.HttpStatus;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.sql.Date;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collection;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+public class WebBannerIntegrationTest extends XAuthIntegrationTest {
+
+    @Autowired
+    private BannerRepository bannerRepository;
+
+    private static final String BANNER_ENDPOINT = "/web/banners/";
+
+    private Banner banner;
+    private Collection<Banner> banners;
+    private User user;
+    private User admin;
+
+    private ObjectMapper mapper;
+
+    @Before
+    public void setUp() {
+        banner = new Banner();
+        banners = new ArrayList<>();
+        banner.setText("AreaFiftyLAN starts in less than a month! Make sure to get your tickets!");
+        banner.setStartDate(Date.valueOf("2017-02-03"));
+        banner.setEndDate(Date.valueOf("2018-02-28"));
+        banners.add(bannerRepository.save(banner));
+
+        user = createUser();
+        admin = createAdmin();
+
+        mapper = new ObjectMapper();
+    }
+
+    @After
+    public void cleanUpBanners() {
+        bannerRepository.deleteAll();
+    }
+
+    //region GET all banners
+    @Test
+    public void testGetBannersWithoutPermissions() {
```

Comment:
Could you please make sure that the naming of the tests is consistent with the rest of the repository? The current scheme is "...AsUser" or "...AsAdmin" instead of "...WithPermissions" etc...

Reply:
👍